richeditor-compose/src/commonMain/kotlin/com/mohamedrejeb/richeditor/paragraph/type/UnorderedList.kt

```diff
@@ -7,16 +7,18 @@ import com.mohamedrejeb.richeditor.annotation.ExperimentalRichTextApi
 import com.mohamedrejeb.richeditor.model.RichSpan
 import com.mohamedrejeb.richeditor.paragraph.RichParagraph
 
-internal class UnorderedList : ParagraphType {
+internal class UnorderedList(
+    private val indent: Int = 38
```

Comment:
The indent param here shouldn't have a default value. It should be passed by the state instead.